src/store/global/global.thunks.ts

```diff
@@ -440,6 +441,69 @@ function checkPendingDelayedWithdrawals(): AppThunk {
   };
 }
 
+/**
+ * Checks if we have some exit that is not saved in localStorage
```

Comment:
It seems like we should just remove the storage for this and depend entirely on this method? And then in the exit component or the getExits thunk we filter between delayed withdraws and normal withdraws. Not sure what the best place would be, maybe @amonsosanz @elias-garcia have thoughts here

Reply:
At this moment we need something while the tx is confirmed in L1 and keep localStorage give us some consistency until find a better solution. I think that we could improve the flow based in the information from the network

Reply:
In general terms, I kind of agree with @AlbertoElias here in the sense that we should not rely on the local-store version of something that we are already obtaining from the API. That said, TBH I don't understand the reasoning from @adrifdez here:

> we need something while the tx is confirmed in L1